app/controllers/idv/link_sent_controller.rb

```diff
@@ -80,12 +80,7 @@ def render_step_incomplete_error
     end
 
     def take_photo_with_phone_successful?
-      document_capture_session_result.present? && document_capture_session_result.success? &&
-        selfie_requirement_met?
-    end
-
-    def document_capture_session_result
-      @document_capture_session_result ||= document_capture_session&.load_result
+      stored_result&.success? && selfie_requirement_met?
```

Comment:
double checked that the store_result is fetched by the IdvSessionConcern as the old code. 👍